aiohasupervisor/models/backups.py

```diff
@@ -135,7 +135,7 @@ class FullBackupOptions(Request):
     name: str | None = None
     password: str | None = None
     compressed: bool | None = None
-    location: set[str | None] | str | None = None
+    location: list[str | None] | str | None = DEFAULT  # type: ignore[assignment]
```

Comment:
Technically, I guess this is a breaking change, but we only introduced `set[]` support with #33 which isn't in a stable release, so changing that is fine.

Reply:
Yea I noted that in the description